.github/scripts/remote-do-rtl-build.sh

```diff
@@ -18,6 +18,13 @@ cd $REMOTE_CHIPYARD_DIR
 ./scripts/init-submodules-no-riscv-tools.sh --skip-validate
 ./scripts/init-fpga.sh
 
+
+# Constellation can run without espresso, but this improves
+# elaboration time drastically
+pushd $REMOTE_CHIPYARD_DIR/generators/constellation
+scripts/install-espresso.sh $RISCV
```

Comment:
What is espresso? Is this like graalvm?

Reply:
https://github.com/chipsalliance/espresso .

Espresso is a logic minimizer that Constellation calls  (through chisel3) to generate routing tables.
If `espresso` is not installed, the generator falls back on a slower implementation.

Reply:
I'm inclined to just add this installation in `build-toolchains-extra.sh` and add a flag to skip this if users don't want it.

Reply:
I'm going to leave it in like this for now, to avoid more churn on the install scripts.

Reply:
Sure. Can you make an issue so this is noted and we can put it into the next release? I get the feeling that majority of people won't have this installed and will complain about slowness

Reply:
Sure, I added an issue. There has been discussion of packaging espresso as a Scala library so it can just be added as a SBT dependency to Chisel, but I don't think there's been any progress on that. 

Also, the performance improvement is only apparent for large many core designs, which are still slow to elaborate due to the cost of elaborating many cores.